[2025-02-23T15:52:38.872+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:52:38.910+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-08T00:00:00+00:00 [queued]>
[2025-02-23T15:52:38.927+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-08T00:00:00+00:00 [queued]>
[2025-02-23T15:52:38.929+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T15:52:39.038+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): third_task> on 2025-02-08 00:00:00+00:00
[2025-02-23T15:52:39.048+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1462) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T15:52:39.050+0000] {standard_task_runner.py:72} INFO - Started process 1485 to run task
[2025-02-23T15:52:39.048+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'third_task', 'scheduled__2025-02-08T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmp4l48h4_t']
[2025-02-23T15:52:39.052+0000] {standard_task_runner.py:105} INFO - Job 39: Subtask third_task
[2025-02-23T15:52:39.115+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.third_task scheduled__2025-02-08T00:00:00+00:00 [running]> on host 2cebbf768e94
[2025-02-23T15:52:39.203+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-08T00:00:00+00:00'
[2025-02-23T15:52:39.214+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:52:39.238+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T15:52:39.239+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo finally, it will be done!!!']
[2025-02-23T15:52:39.250+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T15:52:39.252+0000] {subprocess.py:106} INFO - finally, it will be done!!!
[2025-02-23T15:52:39.252+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T15:52:39.286+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:52:39.287+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=third_task, run_id=scheduled__2025-02-08T00:00:00+00:00, execution_date=20250208T000000, start_date=20250223T155238, end_date=20250223T155239
[2025-02-23T15:52:39.388+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:52:39.409+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:52:39.411+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-24T04:47:14.651+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:47:14.695+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-08T00:00:00+00:00 [queued]>
[2025-02-24T04:47:14.709+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-08T00:00:00+00:00 [queued]>
[2025-02-24T04:47:14.710+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T04:47:14.761+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): third_task> on 2025-02-08 00:00:00+00:00
[2025-02-24T04:47:14.770+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'third_task', 'scheduled__2025-02-08T00:00:00+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpxdv1re6q']
[2025-02-24T04:47:14.777+0000] {standard_task_runner.py:105} INFO - Job 30: Subtask third_task
[2025-02-24T04:47:14.783+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=894) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:47:14.787+0000] {standard_task_runner.py:72} INFO - Started process 917 to run task
[2025-02-24T04:47:14.914+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.third_task scheduled__2025-02-08T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:47:15.056+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-08T00:00:00+00:00'
[2025-02-24T04:47:15.057+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:47:15.081+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T04:47:15.084+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo finally, it will be done!!!']
[2025-02-24T04:47:15.102+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T04:47:15.105+0000] {subprocess.py:106} INFO - finally, it will be done!!!
[2025-02-24T04:47:15.105+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T04:47:15.140+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:47:15.142+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=third_task, run_id=scheduled__2025-02-08T00:00:00+00:00, execution_date=20250208T000000, start_date=20250224T044714, end_date=20250224T044715
[2025-02-24T04:47:15.265+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:47:15.295+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-24T04:47:15.300+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
